packages/api/src/database/connection.ts

```diff
@@ -1,9 +1,11 @@
 import mongoose, { ConnectOptions } from 'mongoose';
 
-const connectDatabase = async (): Promise<typeof mongoose> => {
+const connectDatabase = async (
+  hostOverride?: string
+): Promise<typeof mongoose> => {
   const username = process.env.MONGO_USERNAME || '';
   const password = process.env.MONGO_PASSWORD || '';
-  const host = process.env.MONGO_HOST || '';
+  const host = hostOverride || process.env.MONGO_HOST || '';
```

Comment:
Thinking through it more -- I realized there is probably a good case for overriding env values for different environments: 
- it reduces the risk that devs accidentally run a dangerous command in the wrong environment
- it makes changing the setup for different environments simpler

Ideally we should only override env variables based only on the `NODE_ENV` - and all the logic describing that would be clear and kept in one place (a single conditional statement). Then we can override the `NODE_ENV` in individual package.json script commands.

For now though, I think this is fine. Let's just open another ticket for the above as a refactor. @kristoferlund

Reply:
I have seen this being referenced here and there: https://12factor.net/

Haven't read it myself yet though. One thing they seem to advocate for is maintaining separate complete env files for different environments rather than overriding etc.

Perhaps I'll just remove the override for now because like we discussed - the scripts can be run from inside the Docker container and use the same env as the production environment.